include/fmt/core.h

```diff
@@ -1613,6 +1629,194 @@ inline void print(const S& format_str, Args&&... args) {
       internal::make_args_checked<Args...>(format_str, args...));
 #endif
 }
+
+namespace internal {
+
+template <typename T, typename Char> struct is_string_view : std::false_type {};
+
+template <typename Char>
+struct is_string_view<basic_string_view<Char>, Char> : std::true_type {};
+
+#ifdef FMT_USE_STRING_VIEW
+template <typename Traits, typename Char>
+struct is_string_view<std::basic_string_view<Char, Traits>, Char>
+    : std::true_type {};
+#endif
+
+#ifdef FMT_USE_EXPERIMENTAL_STRING_VIEW
+template <typename Traits, typename Char>
+struct is_string_view<std::experimental::basic_string_view<Char, Traits>, Char>
+    : std::true_type {};
+#endif
```

Comment:
This can be replaced with 

```
template <typename Char>
struct is_string_view<std_string_view<Char>, Char> : std::true_type {};
```
I wouldn't bother with traits at this time since they are not handled elsewhere in {fmt}.

Reply:
Well, I'd create an issue to track it then...
There's a constructor for `internal::basic_string_view` from `std::basic_string<>` with all the template parameters. But `std_string_view` is recognized with only defaults...